src/Translations.php

```diff
@@ -89,6 +90,7 @@ public function init()
             Plugins::EVENT_AFTER_LOAD_PLUGINS,
             function() { 
                 $this->registerAfterLoadEvents();
+                Craft::$app->queue->push(new SyncDataToHubSpotJob());
```

Comment:
did not get you here - are we impling that this needs to run more than once?

As per the code, this will run on all times when the update or install is performed - but it is safe to do that since we follow a check on n8n which checks and updates or creates.

Reply:
LMK if you have more suggestions on the same.

Reply:
Place a log inside this even and checkout if this event only run when intall/update or this is triggerred on every page reload